prody2/__init__.py

```diff
@@ -104,7 +104,7 @@ def addProDyPackage(cls, env, version, default=False):
 
         # configure ProDy to automatically handle secondary structure information
         installCmd.append('python -c "import os; os.environ.setdefault(\'HOME\', \'{0}\')" &&'.format(Config.SCIPION_HOME + os.path.sep))
-        installCmd.append('python -c "import prody; prody.confProDy(auto_secondary=True)" &&')
+        installCmd.append('python -c "import prody; prody.confProDy(auto_secondary=True, verbosity=\'none\')" &&')
```

Comment:
I'd link PRODY the verbosity with Scipion debug mode.

from pyworkflow import Config

Config.debugOn()

Reply:
Not sure how to do this

Reply:
````
prodyVerbosity =  ' none' if not Config.debugOn() else 'whatever_needs_prody_for_debug'

...
...
installCmd.append('python -c "import prody; prody.confProDy(auto_secondary=True, verbosity=\'%s\')" &&' % prodyVerbosity)
````
Aprox!